projects/ngx-d3timeline/src/lib/content/selectors/activity.selectors.ts

```diff
@@ -46,8 +53,39 @@ const selectDraggedToSeries = createSelector(
   valueToSeries
 );
 
+const selectDeltaTime = createSelector(
+  selectTimeOrientation,
+  selectDragEvent,
+  getDeltaTime
+);
+
+const selectCurrentlyDraggedUpdatedStart = createSelector(
+  selectCurrentlyDraggedActivity,
+  activity => activity && activity.updatedStart
+);
+
+const selectCurrentlyDraggedUpdatedFinish = createSelector(
+  selectCurrentlyDraggedActivity,
+  activity => activity && activity.updatedFinish
```

Comment:
as above

Reply:
Its used in `selectActivityUpdatedForDrag`